build/workbox.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs')
+const swBuild = require('workbox-build')
+const config = require('./workbox.config.json')
+
+const updateUrl = (manifestEntries) => manifestEntries.map((entry) => {
+  if (entry.url.startsWith('_gh_pages/')) {
+    entry.url = entry.url.replace(/_gh_pages\//g, '')
+  }
+  return entry
+})
+
+const workboxSWSrcPath = require.resolve('workbox-sw')
```

Comment:
yep workbox-build ships with workbox-sw as a dependency, so I thought it made better sense to just copy it directly from node_modules, that way everytime we bump the version of workbox-build the relevant version of workbox-sw gets copied too.

Also I just realised i'm copying to assets/js/vendor/
should this instead be copied to _gh_pages/assets/js/vendor/? i'm not sure as assets/js will work locally but gh_pages/assets/js would work remotely?